source/data/Symbol_info.rst

```diff
@@ -6,95 +6,156 @@
 Symbol info
 -----------
 
-The `/symbol_info`_ endpoint returns a list of instruments and a set of rules for them. This endpoint is requested
-every hour.
+The `/symbol_info`_ endpoint returns a list of instruments and a set of rules for them.
+This endpoint is requested every hour.
 
-The response body (JSON) must include an ``s`` field. The response header must include the ``Content-Type`` 
-header with the ``application/json`` value.
+The response body (JSON) must include an ``s`` field.
```

Comment:
I suggest to add an intro phrase or header before we jump to response formatting rules and descriptions.
E.g. ## Response fields